gladiator.js

```diff
@@ -0,0 +1,10 @@
+class gladiator(name, weapon){
```

Comment:
- Class names should always be capitalized.
- The class definition does not take any arguments. Its constructor, however, does.